synfig-studio/src/gui/states/state_bone.cpp

```diff
@@ -345,12 +373,18 @@ StateBone_Context::StateBone_Context(CanvasView *canvas_view) :
 	);
 	// 2, default bone width
 	options_table.attach(bone_width_label,
-						 0, 1, 6, 7, Gtk::EXPAND|Gtk::FILL, Gtk::FILL, 0, 0
+						 0, 1, 2, 3, Gtk::EXPAND|Gtk::FILL, Gtk::FILL, 0, 0
```

Comment:
Now I noticed that you're using deprecated Gtk::Table. Could you switch to Gtk::Grid?

One example of changing was done here 52eb2c84

Reply:
Okay! I actually took inspiration from other states. 😅 Will update the code!

Reply:
This issue is still present in current code.

Reply:
I made the changes here. I will do the tweaking needed to make it look neat.